pkg/components/linkerd/component.go

```diff
@@ -41,8 +41,9 @@ func init() {
 }
 
 type component struct {
-	ControllerReplicas int  `hcl:"controller_replicas,optional"`
-	EnableMonitoring   bool `hcl:"enable_monitoring,optional"`
+	ControllerReplicas int    `hcl:"controller_replicas,optional"`
+	EnableMonitoring   bool   `hcl:"enable_monitoring,optional"`
+	PrometheusURL      string `hcl:"prometheus_url,optional"`
```

Comment:
If we disable built-in Prometheus and Grafana, shouldn't this field be required? Or if this is empty `linkerd` just won't use it?

Reply:
It won't send metrics. But it will still function just fine.